.gitignore

```diff
@@ -69,7 +69,8 @@ extensions/extensions-helm/aissemble-spark-operator-chart/values.yaml
 extensions/extensions-helm/aissemble-jenkins-chart/values.yaml
 extensions/extensions-helm/aissemble-mlflow-chart/values.yaml
 extensions/extensions-helm/aissemble-kafka-chart/values.yaml
-extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/values.yaml
+extensions/extensions-helm/aissemble-fastapi-chart/values.yaml
+extensions/extensions-helm/aissemble-quarkus-chart/values.yaml
```

Comment:
I: instead of having each inheriting chart use templating to insert aissemble version, we can do it in the base charts and just inherit the value.